packages/storefront-events-collector/src/handlers/search/searchRequestSentAEP.ts

```diff
@@ -11,12 +11,13 @@ const handler = async (event: Event): Promise<void> => {
     const { searchUnitId, searchInputContext, debugContext, customContext } = event.eventInfo;
 
     let payload: BeaconSchema;
+
+    const searchInputCtx = createSearchInputCtx(searchUnitId as string, searchInputContext);
```

Comment:
why are we casting this as a string? is it not defined in the types anywhere?

Reply:
@grahamcrackers i did not chase it down, my change was moving this existing line from outside the `else` statement